### DISCUSSION TITLE
[Question] - How do you view your dashboard if a service using JobRunr is hosted on any cloud service like AWS?

### DISCUSSION THREAD

**Discussion by sriharigr-threado** [jobrunr/jobrunr#320] [Q&A]
Hello,

I would like to know how I can monitor recurring jobs and scheduled jobs when a service using jobrunr is hosted on cloud.

Thanks

**Comment by rdehuyss** [comment]
There are options to do so here in the discussions.

**Reply by ShubhajitM** [reply]
> 

where is the discussion? can you point it here?

**Reply by ShubhajitM** [reply]
<img width="1436" alt="image" src="https://user-images.githubusercontent.com/100258269/161701823-193e9553-d3f3-45fb-a856-49b944f3783a.png">

Dashboard is throwing 401 in production

**Comment by patkujawa-wf** [comment]
https://github.com/jobrunr/jobrunr/discussions/249 is related
